app/src/main/java/co/nimblehq/chorn/survey/ui/theme/AppStyles.kt

```diff
@@ -1,9 +1,28 @@
 package co.nimblehq.chorn.survey.ui.theme
 
+import androidx.compose.runtime.Composable
 import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.sp
 
 class AppStyles {
-    // Custom styles here
+
+    private val normalText = TextStyle(
+        fontFamily = NeuzeitFontFamily
+    )
+
+    private val normal17Text: TextStyle
+        @Composable
+        get() = normalText.copy(fontSize = 17.sp)
+
+    val whiteNormal17Text: TextStyle
+        @Composable
+        get() = normal17Text.copy(color = Color.White)
+
+    val whiteAlpha3Normal17Text: TextStyle
+        @Composable
+        get() = normal17Text.copy(color = Color.White.copy(alpha = 0.3f))
```

Comment:
I feel like adding description after `normalText` is easy to understand that it extended from the `normalText` style. 🤔  How about this:
```suggestion
    private val normalText = TextStyle(
        fontFamily = NeuzeitFontFamily
    )

    private val normalText17: TextStyle
        @Composable
        get() = normalText.copy(fontSize = 17.sp)

    val normalText17White: TextStyle
        @Composable
        get() = normal17Text.copy(color = Color.White)

    val normalText17White30: TextStyle
        @Composable
        get() = normal17Text.copy(color = Color.White.copy(alpha = 0.3f))
```